src/components/panel/panel.spec.js

```diff
@@ -544,6 +544,20 @@ describe('$mdPanel', function() {
       expect(PANEL_EL).not.toExist();
     });
 
+    it('should close when clickOutsideToClose set to true and ' +
+        'propagateContainerEvents is also set to true', function() {
+          var config = {
+            propagateContainerEvents: true,
+            clickOutsideToClose: true
+          };
+
+          openPanel(config);
+
+          clickPanelContainer(angular.element(document.body));
```

Comment:
Can you move the `angular.element` part to the clickPanelContainer? I don't think it particularly matters for this test, so it seems to fit better in the helper method.

I'm wondering if we want something like `getElement` to make this more flexible.
https://github.com/angular/material/blob/72d068593b7f353d32f8308870d2b0d1c7c759c9/test/angular-material-spec.js#L287